felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -2801,7 +3085,11 @@ func (m *bpfEndpointManager) ensureNoProgram(ap attachPoint) error {
 	}
 
 	// Forget the policy debug info
-	m.removePolicyDebugInfo(ap.IfaceName(), 4, ap.HookName())
+	if m.ipv6Enabled {
+		m.removePolicyDebugInfo(ap.IfaceName(), 6, ap.HookName())
+	} else {
```

Comment:
again, it is not "either or" can be both

Reply:
Same as above.